docs/dotnet.rst

```diff
@@ -0,0 +1,141 @@
+Dotnet
+======
+
+Prerequisites
+~~~~~~~~~~~~~
+   -  Is written with a Linux-compatible .NET desktop application framework compatible such as
+      `GTKSharp <https://github.com/GtkSharp/GtkSharp>`__, `Avalonia
+      UI <https://www.avaloniaui.net/>`__, or
+      `Eto <https://github.com/picoe/Eto>`__
+   -  Has source code hosted on a Git server such as GitHub, GitLab,
+      or Bitbucket
+
+Steps for Packaging
+~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+  The steps below make a couple of assumptions for the sake of example, that may require modification to work for your use case:
+
+  - The application is namespaced with “com.github.[[username]]” for GitHub users.
+    (This can be replaced with any domain that you own, such as another Git server or your own website domain.)
+  - The application uses .NET 7.
+    (It still should be applicable for other versions by modifying the version number)
+
+Installing dependencies
+^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Install Flatpak using the method provided for your distribution
+   `Flatpak - Quick Setup <https://flatpak.org/setup/>`__
+2. Install the FreeDesktop SDKs (When prompted for a version, select the
+   latest available version)
+
+  .. code-block:: shell
+
+        flatpak install flathub org.freedesktop.Platform org.freedesktop.Sdk
+
+3. Install the Flatpak SDK extension for your dotnet version
+   (e.g. `dotnet6 <https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet6>`__, `dotnet7 <https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet7>`__)
+
+  .. code-block:: shell
+
+        flatpak install org.freedesktop.Sdk.Extension.dotnet7
+
+
+Creating the Flatpak
+^^^^^^^^^^^^^^^^^^^^
+
+5.  Create a new folder somewhere different from your existing project
+
+6.  Create a YAML file titled
+    ``com.github.[[username]].[[project-name]].yaml`` with the following
+    example template, replacing the placeholders with the appropriate
+    information: \
+
+  .. code-block:: yaml
+
+      app-id: com.github.[[username]].[[project-name]]
```

Comment:
it's `io.github`. always.

Reply:
I've updated the guide to reference https://docs.flatpak.org/en/latest/conventions.html#application-ids, since that has the bulk of the information users would need to know to determine what the ID should be.